### PR TITLE
Rework complete tutorial

### PR DESCRIPTION
**Pull request by SarahLendle**
I've reworked the complete tutorial so that it fits our guidelines and works properly.

### CODE REVIEW

File: tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md [hunk 1]
```diff
@@ -1,232 +1,295 @@
...
+5. In Jenkins, enter your copied repository URL as **Project Repository** and choose **Save**.
 
-After selecting your region, your account will be automatically set up for development with **Cloud Foundry**.
+    As a result, Jenkins scans your GitHub repository and triggers a build.
 
-![Cloud foundry view](cftrial.png)
+6. To view the status of your build, choose **Open Blue Ocean** from the navigation pane.
 
-In order to deploy applications on **SAP Cloud Foundry** you need to provide the API endpoint and other properties. You can get them by going inside your trial.
+7. Click on your job to see a detailed log of your job as well as a graphical representation of your pipeline.
```
Comment (MichaelCzcz) [inline tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md:212]: "Click your job" or "Choose your job"

File: tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md [hunk 2]
```diff
@@ -1,232 +1,295 @@
...
+[ACCORDION-END]
 
-!![Jenkins System](jenkins-system.png)
+[ACCORDION-BEGIN [Step 5: ](Add production deployment to your CI/CD pipeline)]
 
-On this screen, click **Add Credentials** and enter username, password, as well as an ID for your credentials record.
+Configure the *Production Deployment* stage of your CI/CD pipeline.
 
-You can now leverage this credentials record by adding the credentials configuration property to the `productionDeployment` stage of your `config.yml`.
+1. In your SAP Cloud Platform trial account, click on your subaccount tile.
```
Comment (MichaelCzcz) [inline tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md:226]: Don't say "click on". Say "click the <insert thing to be clicked>".

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address my comments.

File: tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md (hunk 2 above)
Comment (SarahLendle) [reply tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md]: Done.

File: tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md (hunk 1 above)
Comment (SarahLendle) [reply tutorials/cloudsdk-cap-pipeline-model/cloudsdk-cap-pipeline-model.md]: Done.

**Pull request merged**